src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -516,8 +516,8 @@
 
                 /* Prevents from disallowing clicks on links inside toggles */
                 var toggleLinks = document.querySelectorAll('.sf-toggle a');
-                for (var i = 0; i < toggleLinks.length; i++) {
-                    addEventListener(toggleLinks[i], 'click', function(e) {
+                for (var j = 0; j < toggleLinks.length; j++) {
```

Comment:
Good catch actually: There is no reason there is any difference between the two files. The right fix is that it shouldn't have been called within the above loop at the first place.